frontend/src/components/Helper/AsyncCalls.ts

```diff
@@ -29,3 +30,30 @@ export async function querySuggestions(dispatch: Function, query: string) {
 
     dispatch({type: ActionTypes.putSuggestions, suggestions: [/*retrieve suggestions from backend*/]})
 }
+
+/*
+* store selected items in local storage
+*/
+export async function storeItems(items: Item[]) {
+    try{
+        const jsonItems = JSON.stringify(items);
+        await AsyncStorage.setItem('@save_list', jsonItems);
+    } catch (e) {
+        //TODO
+        //could alert user
+    }
+}
+
+/*
+* store selected items in local storage
+*/
+export async function retrieveItems() {
+    try{
+        const jsonItems = await AsyncStorage.getItem('@save_list');
+        const items:Item[] = jsonItems != null ? JSON.parse(jsonItems) : null;
+        return items;
+    } catch (e) {
+        //TODO
+        //could alert user
+    }
+}
```

Comment:
Could you add a new line to the end of this file? It's kinda OCD but I don't like the big red circle that GitHub puts when there isn't a newline

Reply:
Okay I didn't know that